submissions/Snowfall/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "manifest_version": 2,
```

Comment:
your extension must be using manifest version 3

Reply:
I think it'll also be nice if the garlands were lighting up

Reply:
How do you make firefox work with manifest v3 background service workers? I'm not entirely sure how I could do that. Also aren't garlands and lights seperate things? I'll try adding lights to them.

Reply:
Can I have two different versions for chrome and for firefox, one using manifest V3 and the other V2 respectively?